pages/vaults/index.tsx

```diff
@@ -160,20 +160,36 @@ function Index(): ReactElement {
 	const retiredVaults = useFilteredVaults(vaultsRetired, ({address}): boolean => filterMigrationCallback(address));
 
 	const categoriesToDisplay = useMemo((): TListHeroCategory<string>[] => {
-		const categories = [
-			{value: 'All Vaults', label: 'All', isSelected: category === 'All Vaults'},
-			{value: 'Featured Vaults', label: 'Featured', isSelected: category === 'Featured Vaults'},
-			{value: 'Crypto Vaults', label: 'Crypto', isSelected: category === 'Crypto Vaults'},
-			{value: 'Stables Vaults', label: 'Stables', isSelected: category === 'Stables Vaults'},
-			{value: 'Curve Vaults', label: 'Curve', isSelected: category === 'Curve Vaults'}
-		];
+		const categories = [];
+
+		if ([1, 250, 42161].includes(safeChainID)) {
+			categories.push({value: 'All Vaults', label: 'All', isSelected: category === 'All Vaults'});
+			categories.push({value: 'Featured Vaults', label: 'Featured', isSelected: category === 'Featured Vaults'});
```

Comment:
Rather than having all this duplications, can we just push unique vaults or do we want this to push them in some sort of order?